Duplicati/Library/Main/Operation/Backup/StreamBlockSplitter.cs

```diff
@@ -103,13 +103,10 @@ public static Task Run(Options options, BackupDatabase database, ITaskReader tas
                             var lastread = 0;
                             var buf = new byte[blocksize];
                             var lastupdate = DateTime.Now;
-                            var firstBlock = true;
 
                             // Core processing loop, read blocks of data and hash individually
-                            while (((lastread = await stream.ForceStreamReadAsync(buf, blocksize)) != 0) || firstBlock)
+                            while (((lastread = await stream.ForceStreamReadAsync(buf, blocksize)) != 0))

```

Comment:
The `firstblock` was used to ensure that at least one block was emitted, also for empty files.
Now that it is removed, it is possible to have files with zero blocks, and I could imagine that one or more queries fail because of this assumption.

Not sure how to find these though...